### PR TITLE
Remove old obsolete strings from Fluent files

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Removing old (3+ months) obsolete Fluent strings from Fluent files and any mention of them from HTML files (usually placed in `fallback` helpers)

✍🏼 **Things to note:** ✍🏼
- I left the obsolete strings related to the Mozilla accounts update + Monitor updates, since the new strings are still fresh, and we're still using the older strings as fallbacks. We can remove the obsolete strings start of Q1 2024
- I updated WNP 114-119 to include the new title string that was created in a Fluent file but not used. I confirmed with Craig to update the files.

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/13714

## Testing
The following are pages whose HTML files were edited to remove old strings in `fallback`/`ftl_has_messages` helpers
Did I miss anything?

- http://localhost:8000/en-US/firefox/99.0a1/whatsnew/
- http://localhost:8000/en-US/products/vpn/desktop/windows/
- http://localhost:8000/en-US/products/vpn/desktop/
- http://localhost:8000/en-US/products/vpn/desktop/linux/
- http://localhost:8000/en-US/products/vpn/mobile/
- http://localhost:8000/en-US/products/vpn/mobile/ios
- http://localhost:8000/en-US/products/vpn/download/
- http://localhost:8000/en-US/firefox/accounts/
- http://localhost:8000/en-US/firefox/set-as-default/
- http://localhost:8000/en-US/firefox/browsers/compare/
- http://localhost:8000/en-US/firefox/browsers/compare/brave/
- http://localhost:8000/en-US/firefox/browsers/compare/chrome/
- http://localhost:8000/en-US/firefox/browsers/compare/safari/
- http://localhost:8000/en-US/firefox/browsers/compare/edge/
- http://localhost:8000/en-US/firefox/browsers/compare/opera/
- http://localhost:8000/en-US/firefox/browsers/compare/ie/
- http://localhost:8000/en-US/firefox/browsers/mobile/
- http://localhost:8000/en-US/firefox/browsers/mobile/android
- http://localhost:8000/en-US/firefox/browsers/mobile/ios
- http://localhost:8000/en-US/firefox/browsers/mobile/compare/
- http://localhost:8000/en-US/firefox/browsers/mobile/focus/
- http://localhost:8000/en-US/firefox/developer/
- http://localhost:8000/en-US/firefox/facebookcontainer/
- http://localhost:8000/en-US/firefox/features/adblocker/
- http://localhost:8000/en-US/firefox/features/customize/
- http://localhost:8000/en-US/firefox/features/sync/
- http://localhost:8000/en-US/firefox/new/
- http://localhost:8000/en-US/firefox/99.0a1/whatsnew/
- http://localhost:8000/en-US/mission/
- http://localhost:8000/en-US/privacy/faq/

### WNP page title string updates:
- http://localhost:8000/en-GB/firefox/114.0/whatsnew/
- http://localhost:8000/de/firefox/114.0/whatsnew/
- http://localhost:8000/en-US/firefox/114.0/whatsnew/
- http://localhost:8000/fr/firefox/114.0/whatsnew/
- http://localhost:8000/de/firefox/115.0/whatsnew/
- http://localhost:8000/fr/firefox/115.0/whatsnew/
- http://localhost:8000/en-GB/firefox/115.0/whatsnew/
- http://localhost:8000/en-US/firefox/115.0/whatsnew/?geo=gb
-  http://localhost:8000/en-GB/firefox/116.0/whatsnew/
- http://localhost:8000/de/firefox/116.0/whatsnew/
- http://localhost:8000/fr/firefox/116.0/whatsnew/
- http://localhost:8000/en-US/firefox/117.0/whatsnew/
- http://localhost:8000/en-GB/firefox/117.0/whatsnew/
- http://localhost:8000/fr/firefox/117.0/whatsnew/
- http://localhost:8000/de/firefox/117.0/whatsnew/
- http://localhost:8000/en-US/firefox/118.0/whatsnew/
- http://localhost:8000/en-GB/firefox/118.0/whatsnew/
- http://localhost:8000/fr/firefox/118.0/whatsnew/
- http://localhost:8000/de/firefox/118.0/whatsnew/
- http://localhost:8000/en-US/firefox/119.0/whatsnew/
- http://localhost:8000/en-GB/firefox/119.0/whatsnew/
- http://localhost:8000/fr/firefox/119.0/whatsnew/
- http://localhost:8000/de/firefox/119.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/index-account.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 {% extends "firefox/whatsnew/base.html" %}
 
-{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_title %}{{ ftl('whatsnew-page-title-v2' ,fallback='whatsnew-page-title') }}{% endblock %}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/index-account.html:9]: **Nit:** The space is on the wrong side of the comma (for all the WNP titles)

If fixing this fills you with dread we can totally leave it.

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Looks great. Very through 💯 Just needs a rebase to fix the conflicts.

File: bedrock/firefox/templates/firefox/whatsnew/index-account.html (hunk 1 above)
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/whatsnew/index-account.html:9]: > If fixing this fills you with dread we can totally leave it.

Well when you put it like that... 😝 j/k, I'll fix it

**Comment by reemhamz** [comment]
^ Seems like I also committed a file that was updated in this PR: https://github.com/mozilla/bedrock/pull/13900, but it seems very minimal and harmless (since its a small string update) so I'll just proceed and merge this PR

**Review by stephaniehobson — APPROVED** [review]
R+

**Pull request merged**